packages/block-library/src/navigation/style.scss

```diff
@@ -327,107 +314,29 @@
 	.is-responsive {
 		display: none;
 	}
-
-	// Horizontal layout
-	display: flex;
-	flex-wrap: wrap;
-
-	.items-justified-space-between & {
-		flex-grow: 1;
-	}
-}
-
-// Vertical layout
-.is-vertical .wp-block-page-list, // Page list.
-.is-vertical .wp-block-navigation__container,
-.is-vertical .wp-block-navigation__responsive-container-content {
-	display: flex;
-	flex-direction: column;
-	align-items: flex-start;
-}
-
-// Justification.
-// These target the named container class to work even with the additional mobile menu containers.
-.items-justified-center {
-	.wp-block-page-list,
-	.wp-block-navigation__container,
-	.wp-block-navigation__responsive-container-content {
-		justify-content: center;
-
-		// For the overlay.
-		align-items: center;
-	}
-}
-
-.items-justified-right {
-	.wp-block-page-list,
-	.wp-block-navigation__container,
-	.wp-block-navigation__responsive-container-content {
-		justify-content: flex-end;
-
-		// For the overlay.
-		align-items: flex-end;
-	}
 }
 
-.items-justified-space-between {
-	.wp-block-navigation__container,
-	.wp-block-navigation__responsive-container-content {
-		display: flex;
-		justify-content: space-between;
-		flex: 1;
-	}
-}
-
-// Vertical justification.
-// The vertical variant does not support space-between.
-.is-vertical.items-justified-center {
-	align-items: center;
-
-	.wp-block-navigation-item {
-		justify-content: center;
-	}
-
-	.wp-block-page-list,
-	.wp-block-navigation__container {
-		align-items: center;
-	}
-}
-
-.is-vertical.items-justified-right {
-	align-items: flex-end;
-
-	.wp-block-navigation-item {
-		justify-content: flex-end;
-	}
-
-	.wp-block-page-list,
-	.wp-block-navigation__container {
-		align-items: flex-end;
-	}
-}
-
-
 /**
  * Mobile menu.
  */
 
-.wp-block-navigation__responsive-container {
-	display: none;
+// Override default positioning
+.wp-block-navigation.is-responsive-menu-open {
 	position: fixed;
 	top: 0;
 	left: 0;
 	right: 0;
 	bottom: 0;
-	z-index: 1;
-	align-items: flex-start;
-	justify-content: flex-start;
+	// Give it a z-index just higher than the adminbar.
+	z-index: 100000;
+	padding: $grid-unit-40*2 $grid-unit-30 $grid-unit-30;
```

Comment:
Normally I'd appreciate the move to variables. But since this is the `style.scss` file meant for the frontend as well, I don't think it's appropriate. The variables are mainly meant for block UI.